cheatsheets/DotNet_Security_Cheat_Sheet.md

```diff
@@ -609,20 +613,55 @@ public void RemoveAntiForgeryCookie(Controller controller)
 }
 ```
 
-NB: You will need to attach the anti-forgery token to Ajax requests.
+#### Using .NET Core 2.0 or later:
+
+After .NET Core 2.0 it is possible to automatically generate and verify the antiforgery token.
 
-After .NET Core 2.0 it is possible to automatically generate and verify the antiforgery token. Forms must have the requisite helper as seen here:
+If you are using tag-helpers - which is the default for most web project templates - all forms will automatically send the anti-forgery token.
```

Comment:
Kindly link tag-helpers to the following [doc](https://docs.microsoft.com/en-us/aspnet/core/mvc/views/tag-helpers/intro)